src/Input/InputLabel.js

```diff
@@ -12,28 +12,34 @@ export const styleSheet = createStyleSheet('MuiInputLabel', (theme) => {
   return {
     root: {
       transformOrigin: 'top left',
+      fontSmoothing: 'antialiased',
     },
     formControl: {
       position: 'absolute',
       left: 0,
       top: 0,
-      transform: 'translate(0, 18px) scale(1)',
+      transform: 'translate(0, 40px)',
     },
     shrink: {
-      transform: 'translate(0, 0px) scale(0.75)',
+      fontSize: 12,
+      transform: 'translate(0, 18px)',
       transformOrigin: 'top left',
     },
     animated: {
-      transition: transitions.create('transform', {
+      transition: transitions.create(['transform', 'font-size'], {
```

Comment:
Hum, I see two advantages of animating the scale:
1. Better performance
2. Simpler font size override